idx/memory/tag_query_test.go

```diff
@@ -182,33 +183,6 @@ func TestTagExpressionQueryByTagWithFrom(t *testing.T) {
 	}
 }
 
-func TestSingleTagQueryByTagWithFrom(t *testing.T) {
-	tagIdx, byId := getTestIndex(t)
-	memIdx := New()
-	memIdx.Tags[1] = tagIdx
-	memIdx.DefById = byId
-
-	res := memIdx.Tag(1, "key1", 0)
-	if res["value1"] != 4 {
-		t.Fatalf("Expected %d results, but got %d", 4, len(res))
-	}
-
-	res = memIdx.Tag(1, "key1", 2)
-	if res["value1"] != 3 {
-		t.Fatalf("Expected %d results, but got %d", 4, len(res))
-	}
-
-	res = memIdx.Tag(1, "key1", 3)
-	if res["value1"] != 2 {
-		t.Fatalf("Expected %d results, but got %d", 4, len(res))
-	}
-
-	res = memIdx.Tag(1, "key1", 4)
-	if res["value1"] != 1 {
-		t.Fatalf("Expected %d results, but got %d", 4, len(res))
-	}
-}
-
```

Comment:
why is this removed?

Reply:
because it is redundant with the tests in `memory_find_test.go`, such as `TestTagDetailsWithFrom()`